cmd/kogito/command/shared/install_operator.go

```diff
@@ -113,14 +129,18 @@ func installOperatorWithYamlFiles(image string, namespace string, cli *client.Cl
 			return err
 		}
 	}
-
-	if err := decodeAndCreateKubeObject(box, fileServiceAccountYaml, &v1.ServiceAccount{}, namespace, cli, nil); err != nil {
+	if err := decodeAndCreateKubeObject(box, fileNamespaceYaml, &v1.Namespace{}, namespace, cli, nil); err != nil {
```

Comment:
Nitpick - namespace should be created just when `namespaced` is false as otherwise operator is installed directly in user namespace.

Reply:
Changed